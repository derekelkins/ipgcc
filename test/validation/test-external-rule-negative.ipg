// %declare ExternalA ExternalB ExternalC ExternalD ExternalE %end

S -> ExternalA(1, "a")
     { x = ExternalA.z }
     for i = 0 to 3 do ExternalB[i, i+1]
     { y = ExternalB(1).y }
     repeat ExternalC.z
     repeat ExternalD.z until ExternalE;
