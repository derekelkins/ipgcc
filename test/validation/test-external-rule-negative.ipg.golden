Rule ExternalA in rule S is undefined
Rule ExternalA in rule S is undefined
Rule ExternalB in rule S is undefined
Rule ExternalB in rule S is undefined
Rule ExternalC in rule S is undefined
Rule ExternalC in rule S is undefined
Rule ExternalB in rule S is undefined
Rule ExternalD in rule S is undefined
Rule ExternalE in rule S is undefined
Rule ExternalD in rule S is undefined
Rule ExternalC in rule S is undefined